### PR TITLE
Add a test case for sparse logistic regression with standardization per a hanging issue has been fixed in cuml 24.12

### PR DESCRIPTION
**Pull request by lijinf2**
The hanging occurs when only some of all GPUs get sparse matrix of all zeroes.

### CODE REVIEW

**Comment by lijinf2** [comment]
build

File: python/tests/test_logistic_regression.py [hunk 1]
```diff
@@ -2264,3 +2264,48 @@ def test_sparse_all_zeroes(
...
+) -> None:
+    tolerance = 0.001
+
+    if gpu_number < 2:
+        pytest.skip(reason="test_sparse_one_gpu_zeroes requires at least 2 GPUs")
+    gpu_number = 2
+
+    with CleanSparkSession() as spark:
+        data = [
+            Row(label=1.0, weight=1.0, features=Vectors.sparse(2, {0: 10.0, 1: 20.0})),
```
Comment (eordentlich) [inline python/tests/test_logistic_regression.py:2284]: weight column needed?

**Review by eordentlich — APPROVED** [review]
👍

File: python/tests/test_logistic_regression.py (hunk 1 above)
Comment (lijinf2) [reply python/tests/test_logistic_regression.py:2284]: Good catch. Will remove it in the next PR.

**Pull request merged**